HW1/P3/P3.txt

```diff
@@ -1 +1,3 @@
 # Your discussion here
+A Single entry from the RDD
+(u'aerst', 11, [u'arets', u'aster', u'rates', u'reast', u'resat', u'stare', u'stear', u'strae', u'tares', u'tears', u'teras'])
```

Comment:
I got the same answer as you did. I guess there are three words that have 11 anagrams. So any of them should be correct.